S32-list/create.t

```diff
@@ -22,7 +22,17 @@ isa-ok($list_obj, List, 'Creating a new list object with new works.');
 is($list_obj, list(4, 5, 6), 'The list object contains the right values.');
 is(+$list_obj, 3, 'Finding the length functions properly.');
 }
-
+{
+my @list-pos is List = 1, 2, 3;
+isa-ok(@list-pos, List, "List can be created by 'is List'");
+isa-ok(@list-pos, Positional, 'Is Positional');
```

Comment:
I think this test is redundant. `List` is a `Positional` by definition. If there be a test for this assertion then most definitely belongs to another test suite.

Reply:
`List` is a `Positional` by definition, but that "definition" isn't yet tested in roast.
Any way, indeed I think that test belongs to another file. I'll search for a better place.